### DISCUSSION TITLE
springboot mvc example csproj possible?

### DISCUSSION THREAD

**Discussion by jkone27** [ikvm-revived/ikvm-sdk-maven#58] [Ideas]
would be great (if possible?) to have a minimal mvc springboot example.. (if possible)

https://docs.spring.io/spring-boot/docs/current/maven-plugin/reference/htmlsingle/

**Comment by wasabii** [comment]
This is most definately out of scope for this project, at least. I'd be happy to open a repository under ikvmnet for something though, but that something would need to be more clearly defined. I'd move the conversation for this to Discussions also.
